Widget/app/src/main/java/com/microsoft/device/display/samples/widget/feed/RssFeed.kt

```diff
@@ -91,7 +93,7 @@ object RssFeed {
             .client(okHttpClient).build()
     }
 
-    private fun parseXml(xmlString: String): List<RssItem?> {
+    fun parseXml(xmlString: String): List<RssItem?> {
```

Comment:
+ the below comment, I think this should be the function that should be tested, we shouldn't care about the network layer used when we want to test that the parser works as expected.

Reply:
Agreed, fixed